openproblems/tasks/cell_cell_communication/methods/sca.py

```diff
@@ -0,0 +1,21 @@
+from ....tools.decorators import method
+from .liana import liana
```

Comment:
This method can probably just be moved to `liana.py`, and then they can also share a decorator partial.